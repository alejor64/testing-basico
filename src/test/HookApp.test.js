import React from 'react'
import '@testing-library/jest-dom'
import {shallow} from 'enzyme'
import HookApp from '../HookApp'

describe('Pruebas en el componente <HookApp />', () => {
    test('Debe renderizar el componente', () => {
        const wrapper = shallow(<HookApp />)

        expect(wrapper).toMatchSnapshot()
    })
    
})
