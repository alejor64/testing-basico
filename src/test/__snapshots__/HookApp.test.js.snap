// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el componente <HookApp /> Debe renderizar el componente 1`] = `
<div>
  <h1>
    Hola mundo
  </h1>
</div>
`;
